stripper/ze_Mirrors_Edge_Reborn_p3.cfg

```diff
@@ -1,115 +1,232 @@
-		"OnTrigger" "musicadefondogodPlaySound0-1"
-	}
-	insert:
-	{
-		"OnTrigger" "musicadefondogod,PlaySound,,3,1"
-	}
-}
-
-;Spawn start fix.
-modify:
-{
-	match:
-	{
-		"classname" "info_player_terrorist"
-		"origin" "144 1312 16"
-	}
-	replace:
-	{
-		"origin" "144 1312 -24"
-	}
+;set level 2 immediately after triggering the heli on race level, first place people would constantly suicide to stop the level from advancing
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "easybutton"
+	}
+	insert:
+	{
+		"OnPressed" "level1Enable0-1"
+		"OnPressed" "level2Disable0-1"
+		"OnPressed" "LevelCounterSetValueNoFire20-1"
+	}
+}
+
+;Stopmusic fix, music delay.
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "Lvl1"
+	}
+	delete:
+	{
+		"OnTrigger" "musicadefondoeasyPlaySound01"
+	}
+	insert:
+	{
+		"OnTrigger" "musicadefondoeasy,PlaySound,,3,1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "Lvl2"
+	}
+	delete:
+	{
+		"OnTrigger" "musicadefondoPlaySound0-1"
+	}
+	insert:
+	{
+		"OnTrigger" "musicadefondo,PlaySound,,3,1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "Lvl3"
+	}
+	delete:
+	{
+		"OnTrigger" "musicadefondoPlaySound0-1"
+	}
+	insert:
+	{
+		"OnTrigger" "musicadefondo,PlaySound,,3,1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "Lvl4"
+	}
+	delete:
+	{
+		"OnTrigger" "musicadefondoextremePlaySound0-1"
+	}
+	insert:
+	{
+		"OnTrigger" "musicadefondoextreme,PlaySound,,3,1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "Lvl5"
+	}
+	delete:
+	{
+		"OnTrigger" "musicadefondogodPlaySound0-1"
+	}
+	insert:
+	{
+		"OnTrigger" "musicadefondogod,PlaySound,,3,1"
+	}
+}
+
+;Spawn start fix.
+modify:
+{
+	match:
+	{
+		"classname" "info_player_terrorist"
+		"origin" "144 1312 16"
+	}
+	replace:
+	{
+		"origin" "144 1312 -24"
+	}
+}
+
+;february 2021
+;by gxp 
+
+;remove old relay replace with logic_auto for better reason.
+filter:
+{
+	"classname" "logic_relay"
+	"targetname" "Auto"
+}
+
+add:
+{
+	"classname" "logic_auto"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "serverCommandsay >>>Map created by Dark Arrow<<<20-1"
+		"OnMapSpawn" "serverCommandsay >>>Edited first by [pF]Detonator<<<21-1"
+		"OnMapSpawn" "serverCommandsay >>>Finished by Hallucinogenic Troll<<<22-1"
+		"OnMapSpawn" "zm_comandosTrigger0-1"
+		"OnMapSpawn" "TCSetAutoExposureMax1.50-1"
+		"OnMapSpawn" "TCSetAutoExposureMin.40-1"
+		"OnMapSpawn" "TCSetBloomScale.750-1"
+		"OnMapSpawn" "TCSetTonemapRate.20-1"
+		"OnMapSpawn" "env_cascade_lightAddOutputmaxshadowdistance 7680-1"
+		"OnMapSpawn" "ClientAllCommandr_screenoverlay pmaterials/onscreen_water_warp_remove0-1"
+		"OnMapSpawn" "ClientAllCommandr_screenoverlay pmaterials/onscreen_water_warp_remove0.1-1"
+	}
+}
+
+;fix tp boost prevent skip.

```

Comment:
Why do you edit the logic_auto in like 4 different places? Add it once with everything you want in it...